tree/ntuple/v7/test/ntuple_storage.cxx

```diff
@@ -165,3 +165,46 @@ TEST(RPageSinkBuf, Basics)
       ASSERT_EQ(column, next_page->first);
    }
 }
+
+TEST(RPageSinkBuf, ParallelZip) {
```

Comment:
Perhaps we could add a metric to the page sink so that we can verify here that the compression actually took the code path for parallel processing.

Reply:
Thank you, just added this in 822a21b